src/Schema/Directives/Fields/CreateDirective.php

```diff
@@ -40,7 +52,10 @@ function ($root, array $args) {
                     ? reset($args)
                     : $args;
 
-                return MutationExecutor::executeCreate($model, collect($args));
+                $this->db->connection()->beginTransaction();
```

Comment:
Let's use the transaction closure instead

Reply:
transaction with Closure is a void function, How exactly do you imagine this?

Reply:
` $this->db->connection()->beginTransaction(function () use ($model, $args) {
                MutationExecutor::executeCreate($model, collect($args));
         }
  );`

how would you return the populated model from closure?